### PR TITLE
 Adding bobabase.hackclub.com

### PR DESCRIPTION
**Pull request by WolfGamer2**
#  Adding bobabase.hackclub.com

## Description

I am adding this domain for the Boba Drops organizer website dashboard. This is to see the status of the website and the status of the payment.

### CODE REVIEW

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Hii, could you sort the file alphabetically in the file?

**Comment by WolfGamer2** [comment]
Changed it.

File: hackclub.com.yaml [hunk 1]
```diff
@@ -650,6 +650,12 @@ bn:
 boba:
   - type: CNAME
     value: hackclub.github.io.
+
```
Comment (alx-alexpark) [inline hackclub.com.yaml:653]:
```suggestion
```

File: hackclub.com.yaml [hunk 2]
```diff
@@ -650,6 +650,12 @@ bn:
 boba:
   - type: CNAME
     value: hackclub.github.io.
+    
+bobabase:
+  - ttl: 600
+    type: CNAME
+    value: cname.vercel-dns.com.
+
```
Comment (alx-alexpark) [inline hackclub.com.yaml:658]:
```suggestion
```

**Comment by WolfGamer2** [comment]
Fixed it.

**Pull request closed without merging**